python/svix/api.py

```diff
@@ -115,376 +165,806 @@ class MessageAttemptListOptions(ListOptionsDouble):
+        )
+
+    def get(
+        self, app_id: str, msg_id: str, attempt_id: str
+    ) -> t.Union[HTTPValidationError, HttpError, MessageAttemptOut, None]:
+        return get_attempt_api_v1_app_app_id_msg_msg_id_attempt_attempt_id_get.sync(
+            client=self._client,
+            app_id=app_id,
+            msg_id=msg_id,
+            attempt_id=attempt_id,
+        )
+
+    def resend(
+        self, app_id: str, msg_id: str, endpoint_id: str, options: PostOptions = PostOptions()
+    ) -> t.Union[HTTPValidationError, HttpError, MessageAttemptOut, None]:
+        return resend_webhook_api_v1_app_app_id_msg_msg_id_endpoint_endpoint_id_resend_post.sync(
+            client=self._client,
+            app_id=app_id,
+            msg_id=msg_id,
+            endpoint_id=endpoint_id,
+            **options.to_dict(),
+        )
 
     def list_attempted_messages(
         self, app_id: str, endpoint_id: str, options: MessageAttemptListOptions = MessageAttemptListOptions()
-    ) -> ListResponseEndpointMessageOut:
-        with self._api() as api:
-            return api.list_attempted_messages_api_v1_app_app_id_endpoint_endpoint_id_msg_get(
-                app_id=app_id, endpoint_id=endpoint_id, **options.to_dict(), _check_return_type=False
-            )
+    ) -> t.Union[HTTPValidationError, HttpError, ListResponseEndpointMessageOut, None]:
+        return list_attempted_messages_api_v1_app_app_id_endpoint_endpoint_id_msg_get.sync(
+            client=self._client,
+            app_id=app_id,
+            endpoint_id=endpoint_id,
+            **options.to_dict(),
+        )
 
     def list_attempted_destinations(
         self, app_id: str, msg_id: str, options: MessageAttemptListOptions = MessageAttemptListOptions()
-    ) -> ListResponseMessageEndpointOut:
-        with self._api() as api:
-            return api.list_attempted_destinations_api_v1_app_app_id_msg_msg_id_endpoint_get(
-                app_id=app_id, msg_id=msg_id, **options.to_dict(), _check_return_type=False
-            )
+    ) -> t.Union[HTTPValidationError, HttpError, ListResponseMessageEndpointOut, None]:
+        return list_attempted_destinations_api_v1_app_app_id_msg_msg_id_endpoint_get.sync(
+            client=self._client,
+            app_id=app_id,
+            msg_id=msg_id,
+            **options.to_dict(),
+        )
 
     def list_attempts_for_endpoint(
         self,
         app_id: str,
         msg_id: str,
         endpoint_id: str,
         options: MessageAttemptListOptions = MessageAttemptListOptions(),
-    ) -> ListResponseMessageAttemptEndpointOut:
-        with self._api() as api:
-            return api.list_attempts_for_endpoint_api_v1_app_app_id_msg_msg_id_endpoint_endpoint_id_attempt_get(
-                app_id=app_id, msg_id=msg_id, endpoint_id=endpoint_id, **options.to_dict(), _check_return_type=False
-            )
+    ) -> t.Union[HTTPValidationError, HttpError, ListResponseMessageAttemptEndpointOut, None]:
```

Comment:
you are still returning union types, this should return `ListResponseMessageAttemptEndpointOut` or raise an error